caramel/models.py

```diff
@@ -147,7 +147,7 @@ class CSR(Base):
                                  order_by="AccessLog.when.desc()")
     certificates = _orm.relationship("Certificate", backref="csr",
                                      order_by="Certificate.not_after.desc()",
-                                     lazy="subquery",
+                                     lazy="dynamic",
```

Comment:
This turns `.certificates` into a Query object.  The one difference to normal loaders, is that it's always truthy, even when it contains nothing.

https://docs.sqlalchemy.org/en/13/orm/collections.html#dynamic-relationship-loaders